README.md

```diff
@@ -1 +1,95 @@
-# code-buddy-backend
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CODE BUDDY BACKEND
+
+> This project is about creating API end point using Ruby on Rails and deploying the API on heroku. The API has meetings and reservations endpoints. The API is used in our front end React project which is based on an app to book a meeting to learn or for coding session with your buddy. 
+
+
+## The learning objective of this project are
+
+- Create rails app
+- Connect react front end app to rails back end
+- Use API end point to get data
+- Fetch data from the API
+
+## Deployment
+
+- Click [here](https://codebuddy-me.herokuapp.com/) to view the documentation of the Code Buddy API.
+
+
+## Kanban board
+
+- Click this [link](https://github.com/Donard97/code-buddy-backend/projects/1) to view the Kanban board that was used while building this app.
+
+- Click [here](https://user-images.githubusercontent.com/74506933/150403327-e2696816-3a09-41f5-b49f-a012d8cec8bb.png) to view the initial state of our Kanban board.
+
+- At the end of this project, there are two(2) people in the team.
+
+

```

Comment:
Add Frontend repo link

Reply:
Thank you for letting me know. I add it.